src/main/java/screens/Window.java

```diff
@@ -0,0 +1,156 @@
+        scene.getStylesheets().add("layoutstyles.css");
+
+        //Setting title to the Stage
+        stage.setTitle("Main window");
+
+        //Adding scene to the stage
+        stage.setScene(scene);
+
+        //Displaying the contents of the stage
+        stage.show();
+        this.logger.info("the main application has started");
+    }
+
+    /**
+     * Sets up the necessary services.
+     */
+    private void setupService() {
+        this.fileSystem = new FileSystem();
+        this.loggerFactory = new LoggerFactory(fileSystem);
+        this.logger = this.loggerFactory.createLogger(this.getClass());
+    }
+
+    /**
+     * Gets the backlog of this class.
+     *
+     * @return BackLog object.
+     */
+    public static Backlog getBackLog() {
+        if (backLog == null) {
+            return new Backlog();
+        }
+        return backLog;
+    }
+
+    /**
+     * Creates the menu bar with its items.
+     *
+     * @param stage Main stage.
+     * @return
+     */
+    public MenuBar createMenuBar(final Stage stage) {
+        MenuBar menuBar = new MenuBar();
+        Menu menu1 = new Menu("File");
+        MenuItem item1 = new MenuItem("New file");
+        final FileChooser fileChooser = new FileChooser();
+        item1.setOnAction(
+                new EventHandler<ActionEvent>() {
+                    public void handle(ActionEvent event) {
+                        File file = fileChooser.showOpenDialog(stage);
+                    }
+                }
+        );
+        menu1.getItems().add(item1);
+
+        Menu menu2 = new Menu("Tools");
+        MenuItem item2 = new MenuItem("Info");
+        item2.setOnAction(
+                new EventHandler<ActionEvent>() {
+                    public void handle(ActionEvent event) {
+                        Scene scene = new Scene(new InfoScreen(), 200, 200);
+                        Stage newStage = new Stage();
+                        newStage.setTitle("Information screen");
+                        newStage.setScene(scene);
+                        newStage.show();
+                        logger.info("a new window has been opened");
+                    }
+                }
+        );
+        MenuItem item3 = new MenuItem("Console log");
+        item3.setOnAction(
+                new EventHandler<ActionEvent>() {
+                    @Override
+                    public void handle(ActionEvent event) {
+                        getBackLog().show();
+                        logger.info("a new window has been opened");
+                    }
+                }
+        );
+        menu2.getItems().add(item2);
+        menu2.getItems().add(item3);
+        menuBar.getMenus().add(menu1);
+        menuBar.getMenus().add(menu2);
+        return menuBar;
+    }
+
+    /**
+     * The initialization of the game.
+     *
+     * @param args the arguments to run.
+     */
+    public static void main(String[] args) {
+        launch(args);
+    }
+}
```

Comment:
You should inspect your code on checkstyle before creating a PR. Please a newline at the end of this file and the layoutstyles.css file.